openshift_tools/ansible/simplezabbix.py

```diff
@@ -21,9 +21,22 @@
 Ansible Runner and the openshift-ansible zbxapi module.
 """
 
-import ansible.runner
+from collections import namedtuple
+from ansible.parsing.dataloader import DataLoader
+from ansible.vars import VariableManager
+from ansible.inventory import Inventory
+from ansible.playbook.play import Play
+from ansible.executor.task_queue_manager import TaskQueueManager
+from ansible.plugins.callback import CallbackBase
```

Comment:
@joelddiaz I don't seem to be able to find any currently. I  know there is work being done to start documenting "supported" apis, but I'm not sure what the state of that is...